### DISCUSSION TITLE
angular/angular-currency-pipe-formatting-currency-in-angular/

### DISCUSSION THREAD

**Discussion by giscus** [arungudelli/angularjswiki#182] [Announcements]
# angular/angular-currency-pipe-formatting-currency-in-angular/

Angular Currency Pipe is one of the bulit in pipe in Angular used to format currency value according to given country code,currency,decimal,locale information.

https://www.angularjswiki.com/angular/angular-currency-pipe-formatting-currency-in-angular/

**Comment by LyonOconner** [comment]
i'm trying use this for colombia but no working :c

**Reply by arungudelli** [reply]
Are you getting any error?

**Comment by guili244540** [comment]
It do not work for me either.  No error but the value does not shows.

**Comment by yadirac9001** [comment]
Great !!!

**Comment by ChamnabYun** [comment]
Great! It works in <b>Angular: 13.x.x</b> but need some change from "symbol" => "symbol-narrow".
e.g:
from
`{{value|currency:"KHR":"symbol"}}`
to
`{{value|currency:"KHR":"symbol-narrow"}}`

**Reply by smartPYJ** [reply]
this actually helped me
